### PR TITLE
testing/server: add archive route to upload a file to container.

### PR DESCRIPTION
**Pull request by tarsisazevedo**
add missing route to upload a file to container.

### CODE REVIEW

File: testing/server.go [hunk 1]
```diff
@@ -503,6 +506,22 @@ func (s *DockerServer) statsContainer(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *DockerServer) uploadToContainer(w http.ResponseWriter, r *http.Request) {
+	id := mux.Vars(r)["id"]
+	container, _, err := s.findContainer(id)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
```
Comment (fsouza) [inline testing/server.go]: You need to add an empty return here.

Comment (fsouza) [inline testing/server.go]: Could you also please add a test for this case. I know that this test is missing in other handlers, but please add it here x)

Thank you!

Comment (tarsisazevedo) [inline testing/server.go]: done! :)

**Comment by fsouza** [comment]
I'll go ahead and merge it, I believe that the build will be green.

Thanks for contributing!

**Pull request merged**